reference/conanfile/tools/intel.rst

```diff
@@ -0,0 +1,142 @@
+.. _conan_tools_intel:
+
+
+conan.tools.intel
+===================
+
+
+IntelCC
+=========
+
+Available since: `1.41.0 <https://github.com/conan-io/conan/releases>`_
+
+This tool helps you to manage the new Intel oneAPI `DPC++/C++ <https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top.html>`_ and
+`Classic <https://software.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top.html>`_ ecosystem in Conan.
+
+.. warning::
+
+    This generator is **experimental** and subject to breaking changes.
+
+.. warning::
+
+    macOS is not supported for the Intel oneAPI DPC++/C++ (icx/icpx or dpcpp) compilers. For macOS or Xcode support, you'll have to use the Intel C++ Classic Compiler.
+
+.. note::
+
+    Remember, you need to have installed previously the `Intel oneAPI software <https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html#gs.cgeofk>`_.
+
+
```

Comment:
Add a note here about the `CMakeToolchain` and `MSBuildToolchain`, if the user is using any of those, it is not needed to use this tool.